src/index.html

```diff
@@ -2,58 +2,228 @@
+                <button id="doRegister">Registro</button>
+            </form>
+
+            <div class="descriptionP2">
+                <p>Sé parte de nuestra comunidad y fortalece tus ideas y conocimientos.</p>
+            </div>
+        </main>
+
+    </section>
+
+    <section class="profileInformation" id="vero y dian">
+        <div>
+            <div class="littleCircle">
+                <img src="images/foto_perfil_circulo.png" alt="foto de perfil usuario" class="responsive-img photo">
+            </div>
+
+            <div class="personalInformationPerfilUser">
+                <div class="namePerfilUser"><strong class="black-text perfilName little">Taco López<i class="material-icons center editProfileIcon">edit</i></strong></div>
+                <div class="professionDescription">Developer Sr. en Accenture</div>
+            </div>
+        </div>
+    </section>
+    <form>
+        <p>
+            <textarea class="comentUser" name="description" placeholder="Escribe un commit..."></textarea>
+        </p>
+        <div class="right-align">
+            <button class="waves-effect waves-light btn-small publication"><i class="material-icons center">image</i></button>
+            <button class="waves-effect waves-light btn-small imegeOfPersonalCommit"><i class="material-icons right">computer</i>Publicar</button>
+        </div>
+    </form>
+    <div>
+        <div class="informationBox">
+
+            <div class="chip boxStyle">
+                <img src="images/foto_perfil_circulo.png" alt="Contact Person">
+                <p>Taco Perez</p>
+            </div>
+            <i class="fas fa-globe-americas world"></i>
+            <i class="material-icons center points">more_vert</i>
+            <p class="coments">¿Cómo centrar texto en HTML?</p>
+        </div>
+        <div class="punchButtons">
+            <div class="likeButton">
+                <a class="waves-effect waves-light btn-small"><i class="material-icons left like">thumb_up</i></a>
+                <span class="likeCounter">5</span>
+            </div>
+            <div class="commentButton">
+                <a class="waves-effect waves-light btn-small"><i class="material-icons left like">mode_comment</i></a>
+                <span class="commentCounter">2</span>
+            </div>
+        </div>
+    </div>
+    <form>
+        <p>
+            <textarea class="comentUser" name="description" rows="5" cols="10" placeholder="Comentar..."></textarea>
+        </p>
+        <div class="right-align">
+            <button class="waves-effect waves-light btn-small publication"><i class="material-icons center">image</i></button>
+            <button class="waves-effect waves-light btn-small imegeOfPersonalCommit"><i class="material-icons right">computer</i>Comentar</button>
+        </div>
+    </form>
+
+    <!-- *********** Editar perfil   *********** -->
```

Comment:
El tener tanto maquetado fijo, convierte esta app en una NO SPA

Reply:
revisaste el máster @HectorBlisS ? ese lo habíamos dejado sin var y sin comentarios :( y también lo modularizamos D:

Reply:
Si, no entiendo!!, que sobre cual, nos hiciste la revisión del código???

Reply:
@HectorBlisS este es el que mandamos a revisión: https://github.com/DianyelaMaldonado/CDMX009-Social-Network/tree/master/src

Nosotras lo habíamos dejado con let, y está mayormente en inglés, te preguntamos porque se nos hace raro que vemos en la revisión cosas que no estaban dentro del que mandamos en el pull request, ese código que nos revisaste está en otros branch el cuál no habíamos corregido y no mandamos en el pull :( . Si nos puedes ayudar por fa respondiéndonos esa duda estaría genial.

Reply:
Hay algo raro porque estábamos conscientes de tus comentarios y por eso habíamos corregido esos detalles dentro de  Máster. Por favor haznos saber si hay posibilidad de que revises esa parte. Gracias por tu feedback!

Reply:
Compañeras, yo revisé este PR, si no debí ver esto, no debería estar aquí. ;)

Reply:
btw: esto  no es un PR
> @HectorBlisS este es el que mandamos a revisión: https://github.com/DianyelaMaldonado/CDMX009-Social-Network/tree/master/src

Reply:
Ok, entendido y anotado, lamentamos que hayas visto esa parte, tendremos más  cuidado con eso en los próximos proyectos. Thank you!